Account.cs

```diff
@@ -0,0 +1,26 @@
+﻿using System.Collections.Generic;
+
+public class Account
+{
+	private string name;
+	private decimal balance;
+
+	public List<Transaction> transactions { get; }
```

Comment:
It's best practice to declare public variables before private ones (and same goes for methods). And it's best practice to name properties like this starting with an uppercase letter 🙂